src/components/Checkbox/CoreuiCheckBox.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { CoreuiCheckBox } from './CoreuiCheckBox';
+import { FormikFormWrapper } from '../FormikFormWrapper/FormikFormWrapper';
+
+it('should show CoreuiCheckBox', () => {
+  const wrapper = shallow(<CoreuiCheckBox name="bar" id="id-foo" label="foo" />);
+  expect(wrapper.find('#id-foo').length).toBe(1);
```

Comment:
This test  actually test if there is ID so this test pass if i have code
```javascript
<span id="id-foo"/>
```
and actually this test only tests props. This tests need to be rewriten and rethink.